src/components/RestaurantCategoriesContainer.jsx

```diff
@@ -0,0 +1,23 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { loadCategories, setSelectedItem } from '../actions/restaurant';
+import ItemButtons from './ItemButtons';
+
+export default function RestaurantCategoriesContainer() {
+  const dispatch = useDispatch();
+  const { categories, selected } = useSelector((state) => state.restaurant);
+
+  useEffect(() => {
+    dispatch(loadCategories());
+  }, []);
+
+  const handleClick = ({ id }) => {
+    dispatch(setSelectedItem({ categoryId: id }));
+  };
+
+  return (
+    <>
+      <ItemButtons items={categories} onClick={handleClick} selected={selected.categoryId} />
+    </>
```

Comment:
```suggestion
    <ItemButtons items={categories} onClick={handleClick} selected={selected.categoryId} />
```

Fragment는 없어도 될 것 같아요

Reply:
넵 !!